erpc_c/transports/erpc_uart_espidf_transport.cpp

```diff
@@ -89,15 +102,36 @@ erpc_status_t UartTransport::underlyingSend(const uint8_t *data, uint32_t size)
     int sent = uart_write_bytes(m_uart.num, (const char*) data, size);
 
     // not sure we need to do that, especially when TX fifo is 0
-    if (sent != -1) 
+    if (sent != -1)
     {
         TickType_t timeout = (size * 10 * 1000 * 2) / m_uart.config.baud_rate / portTICK_RATE_MS;
-        
-        if (uart_wait_tx_done(m_uart.num, timeout < m_timeout ? m_timeout : timeout) == ESP_OK) 
+
+        if (uart_wait_tx_done(m_uart.num, timeout < m_timeout ? m_timeout : timeout) == ESP_OK)
         {
             return kErpcStatus_Success;
-        }    
+        }
     }
 
     return kErpcStatus_SendFailed;
 }
+
+bool UartTransport::hasMessage(void)
+{
+    /*
+     * It's not possible to know whether we truly have a message without reading a couple of bytes
+     * from the UART that indicate the total size of the message beyond the 4-byte header and it is
+     * inconvenient to do to because higher level code in FramedTransport will call
```

Comment:
Typo : "inconvenient to do to because " -> "inconvenient to do so because " ?

Reply:
Oops! Thanks.  Fixed in e33c414